lib/phx_tools_web/components/layouts/root.html.heex

```diff
@@ -17,7 +17,14 @@
     >
     </script>
   </head>
-  <body class="bg-white antialiased">
+
+  <body class="font-montserrat bg-[#110B33]">
+    <div class="relative flex items-center justify-center lg:mb-[22px] md:mb-[33px] sm:mb-[35px] mt-[5%] px-[30%]">
+      <.link target="_blank" href="https://optimum.ba">
+        <PhxToolsWeb.Components.SharedCompoents.optimum_logo />
```

Comment:
Typo in the module name.
Also, `SharedComponents` should be aliased in `phx_tools_web.ex` so that you don't need to type in the whole name.